plugins/search/api-report.md

```diff
@@ -158,6 +159,13 @@ export const SearchModal: ({
 export interface SearchModalProps {
   // (undocumented)
   open?: boolean;
+  // Warning: (ae-forgotten-export) The symbol "SearchBarBaseProps" needs to be exported by the entry point index.d.ts
```

Comment:
Before merging, I think we should make sure to export this type as well. I'm not sure why the root index.ts in this package enumerates inidividual symbols rather than exporting * from ./components, but you'll want to add this type to that file as well for now

Reply:
https://github.com/backstage/backstage/pull/8580